sycl/include/sycl/ext/intel/esimd/detail/atomic_intrin.hpp

```diff
@@ -13,12 +13,109 @@
 
 // This function implements atomic update of pre-existing variable in the
 // absense of C++ 20's atomic_ref.
+
```

Comment:
I missed that completely the first time, but we have a big problem - those functions (atomic_load, etc.) pollute the user namespace. Please wrap it into namespace sycl::ext::intel::esimd::detail